automation/orchestrator.go

```diff
@@ -109,13 +126,19 @@ func (o *Orchestrator) handleTrigger(ctx context.Context, e event.Event) error {
 		if !ok {
 			return fmt.Errorf("handleTrigger: expected event.Payload to be a workflow.ID: %v", e.Payload)
 		}
-		return o.RunWorkflow(ctx, wid)
+		go func() {
+			if err := o.RunWorkflow(ctx, wid); err != nil {
+				log.Errorf("%s", err)
```

Comment:
not sure how to surface this error, is logging it good enough? probably needs more context then, right?

Reply:
Oof, yeah this is tough. I'm not really sure how all the parts fit together, but ideally we'd have something that gets error events off the bus and does something reasonable with them. This is ok for now, maybe there should also be a TODO about handling asynchronous errors better.

Reply:
Talked to @b5 about this. The conclusion was to actually be logging errors in `log.Debug` unless they are a critical failure. In the case of a run or dry run, the user is going to get output about the state of the run when the transform events are caught by the `Save` command.